modules/linux_vnc/manifests/init.pp

```diff
@@ -0,0 +1,44 @@
+# TODO: see build-puppet/modules/vnc/manifests/init.pp
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class linux_vnc {
+  # include config
+  # include users::builder
+  include linux_packages::vnc_server
+  include roles_profiles::profiles::cltbld_user
+
+  if (lookup('cltbld_user.vnc_password') == '') {
+    fail('No VNC password set')
+  }
```

Comment:
hiera lookups should only be done within profiles and then passed as args to the class

Reply:
Fixed all instances.